<div id=toc></div>

# Table of Contents

- [q-bio.BM](#q-bio.BM) [Total: 1]


<div id='q-bio.BM'></div>

# q-bio.BM [[Back]](#toc)

### [1] [DeepPNI: Language- and graph-based model for mutation-driven protein-nucleic acid energetics](https://arxiv.org/abs/2511.22239)
*Somnath Mondal, Tinkal Mondal, Soumajit Pramanik, Rukmankesh Mehra*

Main category: q-bio.BM

TL;DR: DeepPNI是一个基于深度学习的模型，用于预测蛋白质-核酸复合物中突变引起的结合自由能变化。该模型结合了结构和序列特征，并在大型数据集上表现出优异的预测性能。


<details>
  <summary>Details</summary>
Motivation: 蛋白质-核酸相互作用对细胞功能至关重要，突变可能导致疾病。现有实验技术难以预测突变的影响，因此需要开发一种高效准确的计算方法。

Method: 研究人员构建了一个名为DeepPNI的深度学习回归模型，该模型整合了结构特征（通过edge-aware RGCN编码）和序列特征（使用蛋白质语言模型ESM-2提取）。该模型利用大型数据集（1951个突变）进行训练，并使用五折交叉验证进行评估。

Result: DeepPNI在大型数据集上实现了0.76的平均皮尔逊相关系数（PCC），在蛋白质-DNA和蛋白质-RNA复合物数据集上均表现出一致的性能，并且在外部数据集验证中优于现有工具。

Conclusion: DeepPNI模型提供了一种可靠且通用的方法来预测蛋白质-核酸复合物中突变引起的结合自由能变化，有助于理解突变对疾病的影响，并为药物开发提供支持。

Abstract: The interaction between proteins and nucleic acids is crucial for processes that sustain cellular function, including DNA maintenance and the regulation of gene expression and translation. Amino acid mutations in protein-nucleic acid complexes often lead to vital diseases. Experimental techniques have their own specific limitations in predicting mutational effects in protein-nucleic acid complexes. In this study, we compiled a large dataset of 1951 mutations including both protein-DNA and protein-RNA complexes and integrated structural and sequential features to build a deep learning-based regression model named DeepPNI. This model estimates mutation-induced binding free energy changes in protein-nucleic acid complexes. The structural features are encoded via edge-aware RGCN and the sequential features are extracted using protein language model ESM-2. We have achieved a high average Pearson correlation coefficient (PCC) of 0.76 in the large dataset via five-fold cross-validation. Consistent performance across individual dataset of protein-DNA, protein-RNA complexes, and different experimental temperature split dataset make the model generalizable. Our model showed good performance in complex-based five-fold cross-validation, which proved its robustness. In addition, DeepPNI outperformed in external dataset validation, and comparison with existing tools

</details>
